macosx/kext/AudioDevice.cpp

```diff
@@ -453,6 +454,25 @@ int kXAudioDevice::create_audio_controls(IOAudioEngine *audioEngine)
     }
     kx_lock_release(hw,&hw->dsp_lock,&flags);
     
+    for_each_list_entry(item,&hw->microcodes)
+    {
+        dsp_microcode *m=list_item(item,dsp_microcode,list);
+        if(!m) continue;
+        if(strncmp(m->name,"prolog",KX_MAX_STRING)==NULL)
+        {
+            prolog_pgm=m->pgm;
+            break;
+        }
+    }
+    kx_lock_release(hw,&hw->dsp_lock,&flags);
```

Comment:
There's one kx_lock_release() above. Should remove the first one and keep this one.